### PR TITLE
Forward port release-v2.5 branch

### PR DESCRIPTION
**Pull request by brendarearden**
Due to OOB releases, the release-v2.5 branch and the release-v2.6 branch were out of synch. This pr has the most recent changes from release-v2.5 in it. 

Steps used:
1. `git fetch upstream`
2. `git checkout -b forward-port-releases upstream/release-v2.5`
3. `git pull upstream release-v2.6 --rebase
4. Resolved index.yaml merge conflicts by deleting the index.yaml, running `helm repo index --url assets assets`, moving the resulting index.yaml to the top level directory to replace the deleted one, `git add .` and then `git rebase --continue`
5. After --rebase was completed, deleted the index.yaml and regenerated it by running `helm repo index --url assets assets`, moving the resulting index.yaml to the top level directory to replace the deleted one, `git add .` and then pushed upstream for PR.

### CODE REVIEW

**Review by jiaqiluo — CHANGES_REQUESTED** [review]
Can you remove the directory `packages/rancher-kiali-server` ?

File: index.yaml [hunk 1]
```diff
@@ -1305,6 +1305,38 @@ entries:
...
+      for details.
+    digest: 8f138c96cdd1f12d91cae4f9443057bc6e6f994e3111ad6b0da69cd42a349ec8
+    icon: https://charts.rancher.io/assets/logos/istio.svg
+    keywords:
+    - networking
+    - infrastructure
+    name: rancher-istio
+    urls:
+    - assets/rancher-istio-1.9/rancher-istio-1.9.600.tgz
+    version: 1.9.600
```
Comment (jiaqiluo) [inline index.yaml:1339]: new entry 1/2

File: index.yaml [hunk 2]
```diff
@@ -3126,6 +3158,21 @@ entries:
...
+      catalog.cattle.io/release-name: rancher-webhook
+    apiVersion: v2
+    appVersion: 0.1.1
+    created: "2021-07-16T13:14:38.451034-07:00"
+    description: ValidatingAdmissionWebhook for Rancher types
+    digest: a0ee1c4f1338c8c24f3c5129bb41a67a43bffb13bf28c138266cf58dd28f2ce4
+    name: rancher-webhook
+    urls:
+    - assets/rancher-webhook/rancher-webhook-0.1.100+up0.1.1.tgz
+    version: 0.1.100+up0.1.1
```
Comment (jiaqiluo) [inline index.yaml:3175]: new entry 2/2

**Review by jiaqiluo — APPROVED** [review]
LGMT

**Comment by brendarearden** [comment]
closing this PR in favor of https://github.com/rancher/charts/pull/1367

**Pull request closed without merging**